CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Zappa Changelog
 
+## 0.53.0
+* Deprecated ACME v1 for Lets Encrypt
+* Global black formatting
+* Update & Deploying using a docker container
```

Comment:
```suggestion
* Update & deploy using a docker container
```

Reply:
Should we link to the github issue or blog post for an explanation of the current functionality?

Reply:
sure. Feel free to just make your own MR from this one. This release is mostly your baby :D